.rubocop.yml

```diff
@@ -0,0 +1,17 @@
+inherit_from: .rubocop_todo.yml
```

Comment:
@jeffse no need to nuke and start again - this is all fixable. 
The steps are: 

1. Update your local master branch to be in line with the master branch on this repo 
2. Update your local copy of the foundation-6 branch to be up to date with this repo
3. Rebase your pull request branch on top of the foundation-6 branch and push it back to this PR

Here is the help page on syncing your fork: https://help.github.com/articles/syncing-a-fork/

... but I'll break it down for you, assuming only basic git experience (apologies if that's too low-level):

1. First, make sure you have a branch pointing to this repo: 

  ```
  git remote add upstream https://github.com/sgruhier/foundation_rails_helper
  ```
2. Now update your version of the master branch:
  
  ```
  git checkout master
  git merge upstream/master
  git push
  ```

3. Now update your version of the foundation-6 branch (delete it and fetch it again). You'll probably have to force push when you push:

  ```
  git branch -D foundation-6
  git fetch upstream
  git checkout -b foundation-6 upstream/foundation-6
  git push --force-with-lease
  ```

4. Now rebase your feature on top of the foundation-6 branch. This might seem a bit hairy if you haven't done an interactive rebase before, but there's nothing to be concerned about:

  ```
  git checkout foundation-6-closable-option
  git rebase --interactive foundation-6
  ```

  This should open your editor. You'll want to delete all of the lines except the ones relating to commits which you've created (I think there's only one?). When you save and close that file, the rebase should happen. You might get some conflicts relating to formatting (using the old hash style, using double quotes etc.) but hopefully nothing else. 

5. Once you're happy with your local branch, you can force-push it back to your fork

  ```
  git push origin foundation-6-closable-option --force-with-lease
  ```

### Rescuing from screwups

* If anything goes wrong during the rebase, you can quit it with `git rebase abort`
* If your local state gets messed up, but you haven't pushed anything you can reset it back to the state on your github with e.g. `git reset origin/foundation-6-closable-option`

Let me know how you get on, and I'm very happy to help if you have any problems or questions.